tests/unit_tests/test_wave_06.py

```diff
@@ -143,16 +139,14 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, and that the items that were swapped are not there
+    assert result is True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+
+    assert tai.inventory == [item_a, item_b, item_f]
+    assert jesse.inventory == [item_c, item_e, item_d]
```

Comment:
As above, if we check list equality, then the explicit len check isn't needed. But it would be preferred to follow a pattern more like Wave 3 to not force the swapped items to end up at the end of the inventories.